### ISSUE TITLE
[Feature] Laser offset

### ISSUE THREAD

**Issue by ASleepyCat** [Drewol/unnamed-sdvx-clone#469]
**Is your feature request related to a problem? Please describe.**
Implement a laser offset that is independent of input offset.

**Describe the solution you'd like**
A separate setting for laser offset. The laser offset should offset laser ticks (slams and non-slams) as well as visually offset the laser.

**Describe alternatives you've considered**
Input offset could be modified to also visually offset lasers, but I think input offset should purely act as an input offset and not interact with anything visually.

**Comment by ASleepyCat** [comment]
Yeah I don't know about visual offset, maybe if people are still missing lasers even with laser offset this could be explored.

**Comment by Drewol** [comment]
Separated the input offset to laser and button specific ones in 1f9d672f89eedf8aa8cea71cfb2fcdcd3e8f3434 but that's still visually detached from the actual lasers so I don't think this should be closed yet
